relay-monitors/src/lib.rs

```diff
@@ -176,7 +189,19 @@ pub fn process_check_in(payload: &[u8]) -> Result<Vec<u8>, ProcessCheckInError>
         return Err(ProcessCheckInError::InvalidEnvironment);
     }
 
-    Ok(serde_json::to_vec(&check_in)?)
+    static NAMESPACE: OnceCell<Uuid> = OnceCell::new();
+    let namespace = NAMESPACE
+        .get_or_init(|| Uuid::new_v5(&Uuid::NAMESPACE_URL, b"https://sentry.io/crons/#did"));
+
+    // Use the monitor slug as the routing key hint. This helps ensure monitor
+    // check-ins are processed in order by routing check-ins from the same monitor
+    // to the same place.
+    let routing_hint = Uuid::new_v5(namespace, check_in.monitor_slug.as_bytes());
```

Comment:
This UUID is only unique within a single project, that is, monitors with the same name but from different orgs will get the same routing_hint. That might be good enough for consistency, but should we throw the project ID in here to spread the load more evenly? For example, the name `celerybackend_cleanup` [seems to be a built-in](https://docs.celeryq.dev/en/stable/userguide/configuration.html#result-expires) that's being used a lot, see

```sql
select slug, count(*) from sentry_monitor group by slug order by count desc limit 100;
```

Reply:
Ah yeah, that does sound good to me, I'll update it to handle this. Good call!